src/radio.c

```diff
@@ -0,0 +1,250 @@
+/**
+ * collectd - src/radio.c
+ * Copyright (C) 2016 rinigus
+ *
+ *
+ The MIT License (MIT)
+
+ Permission is hereby granted, free of charge, to any person obtaining
+ a copy of this software and associated documentation files (the
+ "Software"), to deal in the Software without restriction, including
+ without limitation the rights to use, copy, modify, merge, publish,
+ distribute, sublicense, and/or sell copies of the Software, and to
+ permit persons to whom the Software is furnished to do so, subject to
+ the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ * Authors:
+ *	 rinigus <http://github.com/rinigus>
+
+ Radio switches are monitored, as exposed by rfkill interface in Linux
+ 
+ Sysfs description: https://www.kernel.org/doc/Documentation/ABI/stable/sysfs-class-rfkill
+ 
+ 
+**/
+
+#include "collectd.h"
+#include "common.h"
+#include "plugin.h"
+
+#define SYSFS_ROOT "/sys/class/rfkill/"
+#define BFSZ 512
+
+static int num_rfkill = 0;
+
+static int radio_init (void)
+{
+	int status;
+	char filename[256];
+
+	num_rfkill = 0;
+  
```

Comment:
It's invisible in Github, but you have extra spaces on a lot of the blank lines from here on down.